lib/node_modules/@stdlib/_tools/scaffold/math-strided-real-typed-unary/scripts/scaffold.sh

```diff
@@ -140,7 +144,7 @@ this_dir="${base_dir}/@stdlib/_tools/scaffold/math-strided-real-typed-unary"
 wrap="${base_dir}/@stdlib/_tools/repl-txt/wrap-desc/bin/cli"
 
 # Define the unary function alias:
-unary_alias="${alias:1}"
```

Comment:
Why was this changed?

Reply:
You cannot take the alias directly from the package.json scaffold meta data. E.g. `diracDelta` needs to become `ddiracDelta` for doubles or `sdiracDelta` for floats, etc.

Reply:
Okay, I missed the context here, was assuming packages with aliases such as `cbrt` and `exp`. I'll revert this change, thanks!